packages/lit-dev-content/site/docs/templates/directives.md

```diff
@@ -122,6 +122,15 @@ Lit includes a number of built-in directives to help with a variety of rendering
 
   </td>
   <td>Caches rendered DOM when changing templates rather than discarding the DOM.</td>
+  </tr>
+
+    <tr>
+  <td>
+
+  [`keyed`](#keyed)
+
+  </td>
+  <td>Associates a renderable value with a unique key, forcing the DOM to re-render if the key changes.</td>
```

Comment:
Something here has caused this artifact:
<img width="853" alt="Screen Shot 2022-05-20 at 9 22 48 AM" src="https://user-images.githubusercontent.com/15080861/169571121-35b32a9a-699f-420d-9100-796f8344c4ef.png">

The tags look completely balanced so maybe it's spacing?

Reply:
That was some spacing that @arthurevans pointed out. Fixed.